tests/test_features_creator.py

```diff
@@ -1 +1,40 @@
-from features_creator import features_creator
+from features_creator.features_creator import get_matching_column_names
+import pandas as pd
+import pytest
+
+
+def test_get_matching_column_names():
+    """
+    Tests the `get_matching_column_names` function.
+    Verifies that it raises the correct exceptions, works in
+    "normal" usage, and does not return any extra columns.
+    """
+    test_df = pd.DataFrame({
+        "week_payment1": [1, 2, 3],
+        "week_payment2": [1, 2, 3],
+        "week_payment3": [1, 2, 3],
+        "othercolumn": [5, 6, 7],
+        "week_payment_string4": [5, 6, 7]
+    })
+
```

Comment:
This is my suggestion when I review your code. It becomes a bit of hassle to swap the test_df since I have to keep the same name or the tests won't pass. How about we extract the columns name at the beginning? This way, it doesn't matter whatever name I use to test

```
    expected_column_names = test_df.columns.values.tolist()[0:3]
    not_expected_column_names = test_df.columns.values.tolist()[3:]
```

Reply:
Sure! I've added this change